### PR TITLE
ALC-360 - fix initial chart gap

### PR DESCRIPTION
**Pull request by hosainnet**
Fix initial chart gap by using the first transaction timestamp instead of the pool create date for the chart

Before:

Screenshot 2021-05-07 at 01 42 53

After:

Screenshot 2021-05-07 at 01 39 06

- [x] Branch name corresponds to Linear ticket number.
- [x] Added sufficient integration and unit tests.
- [x] Addressed a single concern in the least number of changed lines as
      possible.
- [x] Only fixed/added the functionality in question **OR** address wide-spread
      whitespace/style issues, not both.

Thanks!

### CODE REVIEW

File: src/components/RightPanel/index.tsx [hunk 1]
```diff
@@ -20,9 +20,13 @@ export default function RightPanel({
     <div>
       <PoolHeader poolDetail={poolDetail} />
       <PoolPriceChart
-        poolMetrics={poolMetrics as PoolMetric[]}
+        poolMetrics={poolMetrics}
         pokeWeights={pokeWeights}
-        creationTime={poolDetail.createTime}
+        creationTime={
+          poolMetrics.length == 0
```
Comment (ledes) [inline src/components/RightPanel/index.tsx:26]: if you go to PoolPriceChart there is a  function called `formatCreationTime` where you  could do this operation

Comment (hosainnet) [reply src/components/RightPanel/index.tsx]: Makes sense thanks 👍

File: src/components/PoolPriceChart/index.tsx [hunk 2]
```diff
@@ -115,14 +115,19 @@ export default function PoolPriceChart({
...
   };
 
   const formatXAxis = (timestamp: number) => {
     return DateTime.fromMillis(timestamp).toUTC().toFormat(TIME_LABEL_FORMAT);
   };
 
+  const startTime =
+    poolMetrics.length == 0
+      ? formatCreationTime(creationTime as number)
+      : formatStartEndTime(poolMetrics[poolMetrics.length - 1].timestamp);
```
[the remark this answers is not in the document]
Comment (hosainnet) [reply src/components/PoolPriceChart/index.tsx:129]: Yeah I checked the difference, creation time is in seconds and start/end is in millis which the corresponding methods handle

File: src/components/PoolPriceChart/index.tsx [hunk 3]
```diff
@@ -115,14 +115,19 @@ export default function PoolPriceChart({
     return DateTime.fromSeconds(creationTime).toUTC();
```
[the remark this answers is not in the document]
Comment (hosainnet) [reply src/components/PoolPriceChart/index.tsx:115]: Yeh, in t he line you commented on below

**Review by ledes — APPROVED** [review]

**Pull request merged**